<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionEditorDlg</class>
 <widget class="QDialog" name="SectionEditorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cabRepChk">
       <property name="text">
        <string>Cabecera de Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="footRepChk">
       <property name="text">
        <string>Pie de Report</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>AÃ±adir</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="cabPage">
       <property name="title">
        <string>Cabecera de pagina</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="fistPageChk">
          <property name="text">
           <string>En primera pagina</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="footPageChk">
       <property name="text">
        <string>Pie de pagina</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnEdit">
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="6">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btnDown">
     <property name="text">
      <string>Bajar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnUp">
     <property name="text">
      <string>Subir</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btnBorrar">
     <property name="text">
      <string>Borrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
